src/vs/editor/contrib/gotoSymbol/browser/link/clickLinkGesture.ts

```diff
@@ -106,16 +106,18 @@ export class ClickLinkGesture extends Disposable {
 	public readonly onCancel: Event<void> = this._onCancel.event;
 
 	private readonly _editor: ICodeEditor;
+	private readonly _alwaysFireExecuteOnMouseUp?: boolean;
 	private _opts: ClickLinkOptions;
 
 	private _lastMouseMoveEvent: ClickLinkMouseEvent | null;
 	private _hasTriggerKeyOnMouseDown: boolean;
 	private _lineNumberOnMouseDown: number;
 
-	constructor(editor: ICodeEditor) {
+	constructor(editor: ICodeEditor, _alwaysFireOnMouseUp?: boolean) {
```

Comment:
just fyi that using `_` as first character of arguments suppresses the unused warning